chemqulacs/vqe/vqeci.py

```diff
@@ -320,6 +320,24 @@ def vqe(init_params, cost_fn, grad_fn, optimizer):
     return opt_state
 
 
+def _generate_inital_states(
```

Comment:
The underscore prefix on a function indicates that the developer considers it private. However, as we've done in this case, there may be users who actually import and use this function to create spin-adapted electronic configurations. Wouldn't it be better not to prefix this function name with an underscore?

Reply:
"inital" is a misspelling of "initial."